From b97d32b05a221e16b53c116eee7e0da09172cb39 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 31 Dec 2019 13:21:25 +0700
Subject: [PATCH 179/455] arm64: defconfig: Enable SATA_AHCI and
 SATA_AHCI_PLATFORM

This patch enables config for SATA driver

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/configs/defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 9bc5c52..721ab77 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -371,3 +371,5 @@ CONFIG_RENESAS_MFIS_ECC=y
 CONFIG_USB_MASS_STORAGE=m
 CONFIG_USB_G_SERIAL=m
 CONFIG_USB_ETH=m
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_AHCI_PLATFORM=y
-- 
2.7.4

